src/andromede/libs/pypsa_models/pypsa_models.yml

```diff
@@ -200,4 +254,172 @@ library:
       constraints:
         - id: balance
           expression: p1 = - efficiency * p0
-      objective: expec(sum(marginal_cost * p0))
\ No newline at end of file
+      objective: expec(sum(marginal_cost * p0))
+
+  
+    - id: storage_unit
+     #Model for PyPSA storage unit: https://pypsa.readthedocs.io/en/latest/user-guide/components.html#storage-unit
+      parameters:
+          #- id: control #Parameter not instantiated for now
+          #- id: type #Parameter not instantiated for now
+        - id: p_nom
+          time-dependent: false
+          scenario-dependent: false
+        #- id: p_nom_mod
+        #- id: p_nom_extendable #Parameter not instantiated for now
+        #- id: p_nom_min #Parameter not instantiated for now
+        #- id: p_nom_max #Parameter not instantiated for now
+        - id: p_min_pu
+          time-dependent: true
+          scenario-dependent: false
+        - id: p_max_pu
+          time-dependent: true
+          scenario-dependent: false
+        #- id: p_set #Parameter not instantiated for now
+        #- id: q_set #Parameter not instantiated for now
+        - id: sign #default value = 1
+          time-dependent: false
+          scenario-dependent: false
+        #- id: carrier #Parameter not instantiated for now
+        - id: spill_cost #Parameter not instantiated for now
+          time-dependent: true
+          scenario-dependent: false
+        - id: marginal_cost
+          time-dependent: true
+          scenario-dependent: false
+        #- id: marginal_cost_quadratic #Parameter not instantiated for now
+        - id: marginal_cost_storage 
+          time-dependent: true
+          scenario-dependent: false
+        #- id: capital_cost #Parameter not instantiated for now
+        - id: active
```

Comment:
What is the meaning of this variable (boolean to say if the component is enabled ?) ? Where is it used ?